playbooks/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -1,4 +1,22 @@
 ---
+# Uncomment if you plan on using these features. Doing so is not required,
+# but enables pre-requisites checks that will confirm that the right parameters
+# are set.
+
+#openshift_use_all_in_one_cluster_deployment: False
+#openshift_use_cinder_persistent_volume: False
+#openshift_use_cinder_registry: False
+#openshift_use_kuryr: False
```

Comment:
I'm not sure moving this here makes it clearer, I would prefer to have all the kuryr related vars altogether. Otherwise it is possible that user (unintentionally) skips some important ones. Not sure if the same applies for the others. Probably not (at least for most of them)

Reply:
That's fair - my thinking here was that grouping these feature parameters together would make it easier for a user to see all options for, say, storage or networking, and that the prerequisites playbook would take care of making sure that all the relevant parameters are filled. But I'll move openshift_use_kuryr back down if you think it's clearer!